Library/Homebrew/rubocops/cask/url.rb

```diff
@@ -8,13 +8,13 @@ module Cask
       #
       # @example
       #   # bad
-      #   url "https://example.com/foo.dmg",
-      #     verified: "https://example.com"
+      #   url "https://example.com/download/foo.dmg",
+      #     verified: "https://example.com/download"
```

Comment:
Wrong indentation here.

Reply:
I don't see it?

Reply:
`verified` should be aligned with the `"` above.